crnk-spring/src/main/java/io/crnk/spring/boot/v3/CrnkConfigV3.java

```diff
@@ -49,13 +57,22 @@ public CrnkBoot crnkBoot(SpringServiceDiscovery serviceDiscovery) {
 		CrnkBoot boot = new CrnkBoot();
 		boot.setObjectMapper(objectMapper);
 
+		/**
+		 * Try to get an existing `QuerySpecDeserializer` implementation and set it up.
+		 * Otherwise `DefaultQuerySpecDeserializer` instance used.
+		 */
+		try {
+			QuerySpecDeserializer querySpecDeserializer = this.applicationContext.getBean(QuerySpecDeserializer.class);
```

Comment:
this can be moved into CrnkBoot using ServiceDiscovery instead of applicationContext. This way it also works with CDI, Guava, etc.